frontend/pages/DashboardPage/cards/ActivityFeed/GlobalActivityItem/GlobalActivityItem.tsx

```diff
@@ -882,6 +882,16 @@ const TAGGED_TEMPLATES = {
       </>
     );
   },
+  resentConfigProfileBatch: (activity: IActivity) => {
+    return (
+      <>
+        {" "}
+        resent the <b>{activity.details?.profile_name}</b> configuration profile{" "}
+        to {activity.details?.host_count}{" "}
+        {(activity.details?.host_count ?? 0) > 1 ? "hosts." : "host."}
```

Comment:
This is okay assuming this can be generated only for >= 1 host(s). If it can be generated for 0
hosts, we'd want it to say "0 hosts"